jwql/website/apps/jwql/templates/about.html

```diff
@@ -57,6 +57,9 @@ <h4>Acknowledgements</h4>
         {% endfor %}
         </ul>
 
+        <h4>Other attributions:</h4>
+        The loading animation 'magnify' is provided by <a href="https://loading.io">loading.io</a>
+
```

Comment:
Unrelated thought: should we include an attribution to the Astropy project?

Reply:
@laurenmarietta Good idea!  I was thinking that we could also add an attribution for Ann Fields who helped us envision our logo improvements.  We should open a separate 'update-attributions' PR for these.